orm/internal/testpb/bank_query.proto

```diff
@@ -42,7 +42,7 @@ message ListBalanceRequest {
       // denom specifies the value of the Denom index key to use in the query.
       Denom denom = 2;
     }
-
+    
```

Comment:
I think if you run `make proto-all` now this diff will leave.

Reply:
done